examples/plotting/filtering.py

```diff
@@ -1,3 +1,16 @@
+''' A map representation of unemployment rate in US using the `US_Sates Dataset`_.
+This example demonstrates using IndexFilter, ColorMapper and HoverTool with 
+basic plot elements (`Area_glyphs.patches`). When hovering over the points, 
+the states and it's umemployment rate are shown. 
+
+.. bokeh-example-metadata::
+    :sampledata: US_States
+    :apis: bokeh.models.IndexFilter, bokeh.models.HoverTool, bokeh.models.LinearColorMapper
+    :refs: :ref:`ug_basic_patches` > :ref:`ug_basic_data_IndexFilter`
```

Comment:
correct references in this case (also separate with comma)
```suggestion
    :refs: :ref:`ug_basic_areas_patches`, :ref:`g_basic_data_filtering`
```